zipkin-ui/js/spanCleaner.js

```diff
@@ -123,7 +123,7 @@ function compareShared(left, right) {
   if (left === right) {
     return 0;
   } else {
-    return left ? 1 : -1;
+    return left === true ? 1 : -1;
```

Comment:
nit: I think `left ? 1 : -1` is OK.

And sorry, this is not comments for this line, but I noticed wrong codes in L111 & L112, so I add comments.
I think those code should be the following.

```
L111:  if (typeof(left) === 'undefined') return -1;
L112:  if (typeof(right) === 'undefined') return 1;
```